DSCResources/MSFT_xDnsServerForwarder/MSFT_xDnsServerForwarder.schema.mof

```diff
@@ -0,0 +1,7 @@
+[ClassVersion("1.0.0"), FriendlyName("xDnsServerForwarder")]
+class MSFT_xDnsServerForwarder :  OMI_BaseResource
+{
+    [Key] String IPAddress;
+    [Write] Boolean RemoveAll;
```

Comment:
The reason I added the RemoveAll parameter was to allow you to remove all of the forwarders set on a DNS server without needing to know what they are. If I were to remove this parameter, you would have to know every IP Address that you would want to remove.

With the way forwarders are added to newly built DNS servers, you may not know what forwarders get added automatically. It could be the IPs set as the Primary or Secondary DNS servers on the network adapter. It could be the IPs currently set as forwarders on an existing DNS server in the domain.